### PR TITLE
0.76 migration guide

### PR DESCRIPTION
**Pull request by candida-d**

### CODE REVIEW

File: versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md [hunk 1]
```diff
@@ -1,24 +1,70 @@
...
+  --data '{"method":"eth_getLogs","params":[{"address": "0xE4D5c6aE46ADFAF04313081e8C0052A30b6Dd724", "fromBlock": "207349352", "toBlock": "207349392"}],"id":1,"jsonrpc":"2.0"}'
+```
+
+Manipulate the `fromBlock` and `toBlock` values to find the correct allowed block range.
+:::
 
 ## Before you upgrade
 
-Please read the changelog to see all the changes. Below you can find a list of the changes you must perform before the upgrade to [v0.75.6 ↗](https://github.com/vegaprotocol/vega/releases/tag/v0.75.6) in this document.
+Please read the changelogs to see all the changes. Below you can find a list of the changes you must perform before the upgrade to [v0.75.6 ↗](https://github.com/vegaprotocol/vega/releases/tag/v0.75.6).
```
Comment (daniel1302) [inline versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md:67]: We should update link for the changelog

File: versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md [hunk 2]
```diff
@@ -55,9 +101,15 @@ We recommend checking all of the changes on your own. Follow the below instructi
   - data-node config
 3. Compare the new generated file in the temp location and the old file to see the differences.
 
-
 ## Upgrade your node
 To upgrade the network follow the [protocol upgrade documentation](../how-to/upgrade-network.md).
 
+## After you upgrade
+
+1. There is a new network parameter for `evmbridgeconfigs`. This value is hard-coded as it needs to be available when your node starts up.
```
Comment (daniel1302) [inline versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md:109]: I am not sure if it is important but the network param is called: `blockchains.evmBridgeConfigs`

**Review by daniel1302 — APPROVED** [review]

File: versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md (hunk 2 above)
Comment (candida-d) [inline versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md:109]:
```suggestion
1. There is a new network parameter for `blockchains.evmBridgeConfigs`. This value is hard-coded as it needs to be available when your node starts up.
```

File: versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md (hunk 1 above)
Comment (candida-d) [inline versioned_docs/version-v0.75/node-operators/migration-guides/upgrade-node.md:67]:
```suggestion
Please read the changelogs to see all the changes. Below you can find a list of the changes you must perform before the upgrade to [v0.76.1 ↗](https://github.com/vegaprotocol/vega/releases/tag/v0.76.1).
```

**Pull request merged**